setup.py

```diff
@@ -13,13 +13,14 @@
 
 install_requires = [
     'packaging>=20,<22',
-    "numpy>=1.18.0,<1.20.0;python_version<'3.7'",
-    "numpy>=1.20.0,<2;python_version>='3.7'",
-    'pandas>=1.1.3,<2',
-    'scikit-learn>=0.24,<2',
-    'torch>=1.8.0,<2',
-    'torchvision>=0.9.0,<1',
-    'rdt>=1.2.0,<2.0',
+    "numpy>=1.20.0,<2;python_version>='3.7' and python_version<'3.10'",
```

Comment:
instead of saying `and python_version<'3.10', why don't we just lower the upper bound